src/pages/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -320,6 +320,10 @@ A popup object instance could be also created via `sdk` with:
 
 - [authorize](#authorize)
 
+**Sample app with the popup flow**
```

Comment:
Not sure if this is the right place for the link – we're separating "Available class methods" and the method description/sample. I would add the link to the sample app at the end of this section. The same for Redirect

Reply:
I was actually split between those two places for it. I will try to put it at the end of this section as you are right that separating "Available class methods" and the method description/sample may be disturbing.